client/me/security-u2f-key/index.jsx

```diff
@@ -78,6 +79,12 @@ class SecurityU2fKey extends React.Component {
 					! u2fKeys.length && (
 						<Card>
 							<p>Use a Universal 2nd Factor security key to sign in.</p>
+							{ ! isSupported() && (
+								<p>
+									Looks like you browser doesn't support the FIDO U2F standard yet. Read more about
```

Comment:
This looks good!  Should we also add known working browsers here to make it easy for users?